modules/upload/upload.service.js

```diff
@@ -43,10 +43,9 @@ class UploadService {
     });
   }
 
-  async uploadFiles(files) {
+  uploadFiles = async files => {
```

Comment:
Why have you made this change, it is more convenient to use normal methods instead of properties with an arrow functions?